### PR TITLE
Update doc with TargetDataFilters

### PR DESCRIPTION
**Pull request by fp12**
Additional information regarding TargetDataFilters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2527,8 +2528,37 @@ With the default `TargetActor` classes, `Actors` are only valid targets when the
 
 **[⬆ Back to Top](#table-of-contents)**
 
+<a name="concepts-target-data-filters"></a>
+#### 4.11.3 Target Data Filters
+Using both the `Make GanmeplayTargetDataFilter` and `Make Filter Handle` nodes, you can filter out the players's `Pawn` or select only a specific class. If you need more advanced filtering, you can subclass `FGameplayTargetDataFilter` and override the `FilterPassesForActor` function.
```
Comment (Bit-Rot) [inline README.md:2533]: `Make GanmeplayTargetDataFilter` <-- typo

Comment (Bit-Rot) [reply README.md]: `players's` --> `player's`

Comment (fp12) [reply README.md]: Good catch! Thanks :)

Comment (Bit-Rot) [reply README.md]: sure thing, stranger.  Thanks for helping with the docs :)

**Comment by tranek** [comment]
Thank you for the PR and review!

**Pull request merged**